### PR TITLE
FEAT: add basic player shooting

### PR DESCRIPTION
**Pull request by AntGa**
added despawn radius around character coolio

### CODE REVIEW

File: scripts/bullet.gd [hunk 1]
```diff
@@ -0,0 +1,23 @@
+extends Sprite2D
+
+var velocity = Vector2(1,0)
+var speed = 250
+
+var look_once = true
+
+func _physics_process(delta):
```
Comment (tulza) [inline scripts/bullet.gd]: static type needed

File: scripts/bullet.gd [hunk 2]
```diff
@@ -0,0 +1,23 @@
+extends Sprite2D
+
+var velocity = Vector2(1,0)
+var speed = 250
+
+var look_once = true
```
Comment (tulza) [inline scripts/bullet.gd]: add static type
``` gd
var velocity := Vector2(1,0)
var speed : int = 250

var look_once : bool = true
```

File: scripts/player.gd [hunk 3]
```diff
@@ -1,16 +1,25 @@
 extends CharacterBody2D
-
-
-const SPEED: float = 300.0
-
+const SPEED: float: float = 300.0
```
Comment (tulza) [inline scripts/player.gd]: wtf

**Review by tulza — CHANGES_REQUESTED** [review]
not lgtm

File: scripts/player.gd (hunk 3 above)
Comment (AntGa) [reply scripts/player.gd]: sorry im blind boss

Comment (AntGa) [reply scripts/player.gd]: plz accept my pull request

**Review by tulza — APPROVED** [review]

**Pull request merged**